src/orquestra/sdk/_base/_driver/_models.py

```diff
@@ -431,3 +442,123 @@ class Event(NamedTuple):
 
 
 Section = List[Event]
+
+# --- System Logs ---
+
+
+class SystemLogSourceType(str, Enum):
+    """Types of sources that can emit system logs."""
+
+    RAY_HEAD_NODE = "RAY_HEAD_NODE"
+    RAY_WORKER_NODE = "RAY_WORKER_NODE"
+    K8S_EVENT = "K8S_EVENT"
+
+
+class K8sEventLogContainerInfoStatus(pydantic.BaseModel):
+    state: str
+    reason: Optional[str]
+    message: Optional[str]
+
+
+class K8sEventLogContainerInfo(pydantic.BaseModel):
+    name: str
+    status: K8sEventLogContainerInfoStatus
+
+
+class K8sEventLogPodConditions(pydantic.BaseModel):
+    type: str
+    status: str
+    reason: Optional[str]
+    message: Optional[str]
+
+
+class K8sEventLogContent(pydantic.BaseModel):
```

Comment:
**issue**: I couldn't find an API spec for these fields, it's only specified as an [example](https://github.com/zapatacomputing/workflow-driver/blob/2185489dd31fee86e3953bddb37c1fd3754dcde8/openapi/src/resources/workflow-run-system-logs.yaml#L50). We shouldn't rely on behavior outside of the API contract.

It's also a lot of boilerplate to parse and interpret content of these JSONs. The system logs are meant to be noisy, a low-risk, last-effort place to look for errors. I'd suggest we don't even attempt to interpret them and return to the user whatever Platform sent us.

Code is liability; the less code we have to maintain the better!

Reply:
> I'd suggest we don't even attempt to interpret them and return to the user whatever Platform sent us.

By "them" I mean the content of a `K8S_EVENT` JSON object.